src/test/java/com/example/tour/place/PlaceServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.example.tour.place;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class PlaceServiceTest {
+
+    @Autowired
+    private PlaceService placeService;
+
+    private Place place = new Place();
```

Comment:
You'll need to set this before each test, I think. Not at the creation of the PlaceServiceTest class.
Check @BeforeEach from JUnit

https://howtodoinjava.com/junit5/before-each-annotation-example/

Reply:
Otherwise, just let private Place place;
And initialize it in every test. Or just define it locally for each one. Choose one option
(Remember each test must be independent)